pkg/controller/dataimportcron-controller.go

```diff
@@ -730,6 +991,22 @@ func addDataImportCronControllerWatches(mgr manager.Manager, c controller.Contro
 	); err != nil {
 		return err
 	}
+
+	if err := c.Watch(&source.Kind{Type: &cdiv1.StorageProfile{}},
+		handler.EnqueueRequestsFromMapFunc(mapStorageProfileToCron),
+		predicate.Funcs{
+			CreateFunc: func(event.CreateEvent) bool { return false },
```

Comment:
are you sure create is not interesting?

Reply:
yep it is, for example, new storage class that is default being created. thx!